.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
```

Comment:
Ensure that secrets like GITHUB_TOKEN and OPENAI_API_KEY are not hard-coded or exposed in version control. Consider using environment variables or secrets management tools to store them securely.